src/react/sebs_snacks/src/components/recipePage.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from "react";
+import Navbar from "./navbar";
+import { useParams } from "react-router-dom";
+
+//ADD CONTEXT WHEN WRITING CONSOLE LOGS. Ex: 'New Ingredient Object: { "newIngredient": "pepper" }'
+
+/*
+import "./images/carbonara.jpg";
+import "./images/spaghetti.jpg";
+import "./images/pestoPasta.jpg";
+*/
+import "../stylesheets/recipePage.css";
+
+export default function RecipeData(){
+    let params = useParams(); //Gets the recipe from the routing URL, allows us to match the object data to the page
+    let recipeName = params.recipeID;
+    console.log(`Loaded Recipe:${recipeName}`);
+
+    //Handles the states for adding new Ingredient/Instruction
+    const [newIngredient, setNewIngredient] = React.useState(''); //Stores information typed in input box
+    const [newInstruction, setNewInstruction] = React.useState('');
+
+    //Handles states for loading page
+    let [recipe, setRecipe] = React.useState();
+    
+
+    useEffect(() => {
+        const loadRecipe = () => {
+            fetch(`http://localhost:3001/api/recipe/${recipeName}`)
+            .then(res => res.json())
+            .then(jsondata => setRecipe(jsondata))          
+        }
+        loadRecipe();
+    },  []) //Empty brackets means run when page first loads
+
+    return (
+        <div>
+            <Navbar />
+            <body>
+                <br/>
+                    {recipe ? (
+                        <div>
+                            {console.log(recipe)}
```

Comment:
In final submissions, its best to remove any console.logs, as they can contain sensitive information, make it harder to spot warnings/errors or even just slow down the app overall. They are fine for this project tho.